src/__tests__/documenter.test.ts

```diff
@@ -2,9 +2,16 @@
 // SPDX-License-Identifier: Apache-2.0
 import { expect, test } from "vitest";
 
-// @ts-expect-error no types here
-import apiDocs from "../../lib/components/internal/api-docs/components";
+import componentDefinitions from "../../lib/components/internal/api-docs/components";
```

Comment:
decided to follow how we define the documenter tests everywhere even if this package only contains one component in case we add more in the future. 

The old snapshot of the documenter was also flat and does not validate the output, this makes it better.

Reply:
thanks